synced_versions_formulae.json

```diff
@@ -44,7 +44,6 @@
   ["moreutils", "sponge"],
   ["mupdf", "mupdf-tools"],
   ["mysql", "mysql-client"],
-  ["nifi-registry", "nifi"],
```

Comment:
why are these no longer synced?

Reply:
if we sync, currently, there is no way to autobump two formulae, that is why

Reply:
@chenrui333 do they still need to be in sync? CC @Homebrew/core thoughts

Reply:
@fxcoudert Just to clarify: are you saying it's fine to not sync or that these should be synced, I wasn't sure.

@chenrui333 I think it would be ideal to extend the autobump tooling to bump both of these synced formulae in the same PR.

Reply:
I also think the latter is the prefered option. I think they should be synced, formulae from the same codebase like `docker` and `docker-autocomplete` don't have a technical dependency, but it's clear to the public that they should be updated together, that's what the sync list is for.

Reply:
until we have a better strategy of handling multiple updates in one PR, do you folks feel like it would be better to unsync so that they can be bumped at the same time?

(I would want to apply unsync all the other formulae, which would fit into the similar situations)